modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -459,4 +459,97 @@ definition {
 		}
 	}
 
+	@description = "LPS-173894 - Verify that after updating a system object entry, triggered email notifications using Object Author and Current User terms are filled correctly for the To, From Address, From Name, CC, BCC and Template Body fields"
+	@priority = 5
+	test CanUseObjectAuthorAndCurrentUserTermsInEmailNotificationOnSystemObject {
+		property custom.properties = "feature.flag.LPS-173894=true";
```

Comment:
Since this is a priority 5 test you need add the "_property portal.acceptance = "true";_" too. And change the LPS from the feature flag to "_LPS-171625_", this that you're using is from another story.